src/game/gameTimer.js

```diff
@@ -1,11 +1,8 @@
-import { finishGame } from "./gameResult";
-
 // container class where will be displayed timer. Change classname if it is necessary
-const TIMER = document.querySelector(".timer");
-
 let gameTime;
 
-export function setTimer(time = 10) {
+function setTimer(time = 10, finishGame) {
```

Comment:
What I had on my mind was that using a full name would be enough. In shortcut, not `cb` but `callback` and info for the future that when passing, try to name things with their full name and give them names that are verbose enough.

IMO `callback` and `finishGame` both are good in this case.

Reply:
if both cases are good i would like to stay with finisGame now